src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -74,16 +74,19 @@ void OVSetPropComplieModleGetPropTests::SetUp() {
 }
 
 TEST_P(OVEmptyPropertiesTests, SetEmptyProperties) {
+    core->get_versions(target_device);
     OV_ASSERT_NO_THROW(core->get_property(target_device, ov::supported_properties));
     OV_ASSERT_NO_THROW(core->set_property(target_device, AnyMap{}));
 }
 
 // Setting correct properties doesn't throw
 TEST_P(OVPropertiesTests, SetCorrectProperties) {
+    core->get_versions(target_device);
     OV_ASSERT_NO_THROW(core->set_property(target_device, properties));
 }
 
 TEST_P(OVPropertiesTests, canSetPropertyAndCheckGetProperty) {
+    core->get_versions(target_device);
```

Comment:
why do we need it here? Even if we set incorrect properties, `get_property` in the loop creates plugin and can receive exceptions